spec/lib/sufia/user_stat_importer_spec.rb

```diff
@@ -169,6 +211,7 @@
           expect(UserStat.count).to eq 8 # 2 users for 4 days
           UserStat.all.each do |stat|
             expect(stat.file_views).not_to eq 0
+            #expect(stat.work_views).not_to eq 0
```

Comment:
Ack, reverting problem again. Guess it is better to do a diff so I don't over-revert!